README.md

```diff
@@ -173,6 +174,40 @@ class { 'sensu':
 }
 ```
 
+### Installing Plugins
```

Comment:
Need a similar section for Managing Extensions

Reply:
Added section.